collect_app/src/main/java/org/odk/collect/android/backgroundwork/FormUpdateScheduler.java

```diff
@@ -2,7 +2,7 @@
 
 public interface FormUpdateScheduler {
 
-    void scheduleUpdates();
+    void scheduleUpdates(String s);
```

Comment:
Why `s` not `projectId`?

Reply:
Ooops. Probably added this with refactoring tools but forgot to rename.